core/domain/story_domain.py

```diff
@@ -627,6 +630,15 @@ def validate(self):
             raise utils.ValidationError(
                 'Invalid language code: %s' % self.language_code)
 
+        if not isinstance(self.belongs_to_topic, basestring):
+            raise utils.ValidationError(
+                'Expected belongs_to_topic should be a string, received: %s' % (
+                    self.belongs_to_topic))
+
+        if len(self.belongs_to_topic) != 12:
```

Comment:
@seanlip, Can you give your suggestion on these questions:

- Do we need to expose belongs_to_topic value to domain layer as we don't expect users to change this value?
- If yes to the above question, then do we have to validate the id? (note we do not validate the story_id)

@aks681, is there any specific reason for using created_on and  last_updated in the domain layer?

Reply:
"belongs_to_topic" sounds like a boolean. Maybe corresponding_topic_id or something like that.

Yes, expose to domain layer. Domain layer is where the logic is.

No need to validate topic ID.

Reply:
I remember using collections as a reference when creating the new structure models and it had those 2 fields. So, I added that here as well. 
I don't think you need to validate the existence of the topic, though you can make sure it's a 12 character string. I think the former is usually done in the controller layer. What do you think?

Reply:
Yeah, I've done the validation check for the existence of topic in the service layer. Removed the simple validation check for the topic id from the domain layer.